cirq-google/cirq_google/serialization/common_serializers.py

```diff
@@ -41,27 +41,19 @@
 PHASE_MATCH_PHYS_Z = 'phys_z'
 
 
-# Default tolerance for differences in floating point
-# Note that Google protocol buffers use floats
-# which trigger a conversion from double precision to single precision
-# This results in errors possibly up to 1e-6
-# (23 bits for mantissa in single precision)
-_DEFAULT_ATOL = 1e-6
-
-
-def _near_mod_n(e, t, n, atol=_DEFAULT_ATOL):
+def _near_mod_n(e, t, n, atol=fsim_gate_family._DEFAULT_ATOL):
```

Comment:
If we are using this outside of the class, we should change it to not be a private (underscore prefix) variable.